samples/openapi3/client/extensions/x-auth-id-alias/python-experimental/setup.py

```diff
@@ -22,10 +22,10 @@
 # http://pypi.python.org/pypi/setuptools
 
 REQUIRES = [
-  "urllib3 >= 1.15",
-  "certifi",
-  "python-dateutil",
-  "nulltype",
+    "urllib3 >= 1.15",
+    "certifi",
+    "python-dateutil",
+    "nulltype",
```

Comment:
Any chance of moving your black updates into a separate PR?
It will make it easier to review.

Reply:
Yes - black updates were accidental, will pull those out.

Reply:
Reverted.